specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -330,6 +327,10 @@ an exception to this in order to accommodate interface type dependencies—for e
 the Jakarta™ Enterprise Beans SessionContext dependency on the
 _jakarta.xml.rpc.handler.MessageContext_ type.]
 
+If a container provides a newer version of the Java SE platform than 
```

Comment:
Just slight rewording...

`If a container supports a newer version of the Java SE platform than 
Java SE 8, all classes and interfaces provided by the container to satisfy the specification requirements
listed above must be compiled with the Java SE 8 source and class level.`

Reply:
Thanks; done.